### ISSUE TITLE
[BUG_SP]No path found using 2 overloaders while chopping corn

### ISSUE THREAD

**Issue by Martin-L45** [Courseplay/Courseplay_FS22#3239]
**READ FIRST**
https://github.com/Courseplay/Courseplay_FS22/wiki

**Steps to reproduce**
Steps to reproduce the behavior.
This might be more of a enhancement than a bug.
Using 2 overloaders on field 32 on elmcreek. I use the tac 2100 from modhub as overloader.

**Actual behavior**
A clear and concise description of what happen.
This issue does not happen every time. but it would be cool it it can be fixed.
Overloader 1 is idle near trailer.
Overloader 2 gets full behind chopper
overloader 1 tries to create path to chopper but overloader is 2 is blocking where it needs to go. Get message no path found and disables courseplay 

**Expected behavior**
Find a path to the chopper (as it does most of the time)

**Screenshots**
![Farming Simulator 22 Screenshot 2024 05 09 - 19 15 26 84](https://github.com/Courseplay/Courseplay_FS22/assets/85753728/71d8cecf-c575-4f71-9d60-f268c1f70bd2)

**Courseplay version**
Version used: 
7.4.1.2

**Log**
Please attach a log with all debug channels active.
For help with the debug channels,
please check out the readme at the github front page.
[log.txt](https://github.com/Courseplay/Courseplay_FS22/files/15265455/log.txt)

**Map**
Map played on.
Elmcreek

**Vehicles/ implements involved**
Which vehicle or implement?
Mod yes/no ?
Tac2100 from modhub

**Relevant script mods**
Mods like Autodrive or VCA and so on.
EasyDevControls

**Comment by Tensuko** [comment]
It only happens when he have to drive to the chopper?
What chopper do you use?

**Comment by Martin-L45** [comment]
Yes only when he drives to the chopper. I use the ingame John Deere 9000 Series.

**Comment by Tensuko** [comment]
Oh okay so the 1st one wants to go to the chopper but the 2nd one is still at the chopper blocking it.
Hmm yeah thats rather enhancement I think. Will have a look.

**Comment by rohne83** [comment]
Can best be reproduced in first headland. There's high chance that the area where the next unloader shall go to is always blocked for a while, and thus he gets disengaged immediately

**Comment by Crinouche** [comment]
Hello,

I have the same problem, too.
If I simply use trailers that follow the forage harvester, the problem will only arrived on the first turn.
But, if I use the TAC 21000 overloader (as in the photos of Martin-L45) the problem is systematic.
The 1st overloader which is close to the dump trailers will not find the way

**Comment by coatsy35** [comment]
To add to this when you get around to it, I think it needs a  bit more width away from the chopper with wider headers. This is not limited to the first time around the field. I often find the unloader gets stuck behind the header and won't move froward and away to allow the next unloader alongside. see image below, I came back this having sat there for probably 1/2 hour whilst I had to nip out, and have had the same multiple times. Could we have an option to allow the unloader to sit outside the overall width of the header like you would do IRL instead of the default being close to the chopper, that is not something that is done done with the amount of blow choppers have, and I've done enough IRL silage unloading and chopper driving to know 🤣

![fsScreen_2024_08_31_22_30_28](https://github.com/user-attachments/assets/abfbba16-23e2-4dfa-a2fc-9f458d40070c)
